owncloudDomain/src/main/java/com/owncloud/android/domain/spaces/usecases/GetPersonalSpaceForAccountUseCase.kt

```diff
@@ -0,0 +1,36 @@
+/**
+ * ownCloud Android client application
+ *
+ * @author Juan Carlos Garrote Gascón
+ *
+ * Copyright (C) 2023 ownCloud GmbH.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.owncloud.android.domain.spaces.usecases
+
+import com.owncloud.android.domain.BaseUseCase
+import com.owncloud.android.domain.spaces.SpacesRepository
+import com.owncloud.android.domain.spaces.model.OCSpace
+
+class GetPersonalSpaceForAccountUseCase(
```

Comment:
For only accessing to a method of a repository, I am not sure that we need an UseCase.

Reply:
You are right, but in ownCloud we follow the clean architecture structure always and I decided to do it like that to keep consistency, although it is not necessary